firmware/quadruna/VC/src/app/states/app_driveState.c

```diff
@@ -168,6 +174,9 @@ static void driveStateRunOnExit(void)
     // Disable buzzer on exit drive.
     io_efuse_setChannel(EFUSE_CHANNEL_BUZZER, false);
     app_canTx_VC_BuzzerOn_set(false);
+
+    app_canTx_VC_RegenEnabled_set(false);
+    app_canTx_VC_TorqueVectoringEnabled_set(false);
```

Comment:
I believe `VC_TorqueVectoringEnabled` is also not populated anywhere?

Reply:
It's in the TV init function